components/nds-panel/src/Panel.js

```diff
@@ -1,28 +1,20 @@
 import React from "react";
+import classnames from "classnames";
 import PropTypes from "prop-types";
 
 import "./../scss/panel.scss";
 
 export const Panel = props => {
 	const { children, variant, className, ...rest } = props;
 
-	const panelClasses = ["panel"];
-
-	switch (variant) {
-		case Panel.variant.impact:
-			panelClasses.push("panel--impact");
-			break;
-		case Panel.variant.primary:
-			panelClasses.push("panel--primary");
-			break;
-		default:
-			break;
-	}
-
-	if (className) panelClasses.push(className);
+	const classes = classnames([
+		"panel",
+		className,
```

Comment:
I think the appended className should come after the variant, so you can override vertical margin for example